lib/avo/base_resource.rb

```diff
@@ -310,8 +314,8 @@ def fetch_cards
       end
 
       # def action / def filter / def scope
-      define_method entity do |entity_class, arguments: {}|
-        entity_loader(entity).use({class: entity_class, arguments: arguments})
+      define_method entity do |entity_class, arguments: {}, icon: "play"|
```

Comment:
Do we need to specify a default here as well? Wouldn't the initializer default take precedence?

Reply:
Yes, we must specify a default value. I tried as below and it won't get a default icon value.👇
```ruby
define_method entity do |entity_class, arguments: {}, icon: nil|
```